docuware-connector/src/com/axonivy/connector/docuware/connector/DocuWareService.java

```diff
@@ -55,4 +93,126 @@ public Date stringToDate(String dateString) {
 		}
 		return date;
 	}
+	
+	public static Document uploadFile(WebTarget target, java.io.File file, DocuWareEndpointConfiguration configuration, List<DocuWareProperty> properties) throws IOException, DocuWareException {
+		byte[] bytes = Files.readAllBytes(file.toPath());
+		return uploadStream(target,bytes, file.getName(), configuration, properties);
+	}
+	
+	public static Document uploadStream(WebTarget target, ch.ivyteam.ivy.scripting.objects.List<Byte> fileBytes, String fileName, DocuWareEndpointConfiguration configuration, List<DocuWareProperty> properties) throws IOException, DocuWareException {
+		Byte[] bytes = fileBytes.toArray(new Byte[fileBytes.size()]);
+		byte[]  byteArray =  ArrayUtils.toPrimitive(bytes);
+		
+		return uploadStream(target, byteArray,  fileName,  configuration,  properties);
+	}
+	
+	public static Document uploadStream(WebTarget target, byte[] file, String fileName, DocuWareEndpointConfiguration configuration, List<DocuWareProperty> properties) throws IOException, DocuWareException {
```

Comment:
I hope we can soon provide 'inputStream/byte' support directly on the inscription-mask, and then get rid of this file-upload. 
Until then, this serves as great documentation for the circumstance. Thanks